pkg/api/pluginproxy/ds_auth_provider.go

```diff
@@ -49,6 +48,9 @@ func ApplyRoute(ctx context.Context, req *http.Request, proxyPath string, route
 		req.URL.Host = routeURL.Host
 		req.Host = routeURL.Host
 		req.URL.Path = util.JoinURLFragments(routeURL.Path, proxyPath)
+	} else {
+		// fix https://github.com/grafana/grafana/issues/80856 when we skip URL then /{{route.Path}} shall be deleted to avoid error from HTTP severs which not allow dynamic HTTP handlers
```

Comment:
Nit: no need to be that specific:

```suggestion
		// If the route URL is empty, we still need to update the request path with the proxy one.
```